### ISSUE TITLE
Can't see a way to set the dome light position randomly

### ISSUE THREAD

**Issue by ArghyaChatterjee** [owl-project/NVISII#169]
Hello, 

I am trying to follow your NVISII API to set different things inside the DOPE (Deep Object Pose Estimation) training pipeline. Here is the pipeline [script](https://github.com/NVlabs/Deep_Object_Pose/blob/master/scripts/nvisii_data_gen/single_video_pybullet.py). Now I want to set the dome light position randomly but when I try to do that in this way:
```
 # Enable the dome light with the selected position
    light_position = random.choice(dome_light_positions)
    visii.enable_dome_light()
    visii.set_dome_light_position(light_position)

    # Randomly change the position of the dome light for each sample
    light_position = visii.vec3(
         random.uniform(0, 2),
         random.uniform(0, 2),
         random.uniform(0, 2)
     )

    visii.set_dome_light_position(light_position)
   ```
  The python compiler tells me that there is no api like set_dome_light_position. Now how can I set the light position randomly ??

**Comment by TontonTremblay** [comment]
You can only rotate the dome light, you cannot move it sadly.

**Comment by natevm** [comment]
Yeah, dome lights have no position, since they’re meant to model the ambient light from the sky, which only rotates but does not translate.
